src/main/java/mobi/appcent/helium/response/hotspot/HotspotsResponse.java

```diff
@@ -0,0 +1,17 @@
+package mobi.appcent.helium.response.hotspot;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import mobi.appcent.helium.model.hotspot.Hotspot;
+
+import java.util.List;
+
+/**
+ * Created by erenalpaslan on 27.10.2022
+ */
+@Data
+@AllArgsConstructor
+public class HotspotsResponse {
```

Comment:
Is it possible to use a base response class for pageable data something like below:

```
class PageableResponse<T> {
   public T data;
   public String cursor;
}
```